SQL_agg_functions/AdventureWorks-for-Postgres/sql-scripts/select.sql

```diff
@@ -0,0 +1,109 @@
+--1. Identify Department Groupings
```

Comment:
can you please add screenshots with query run and output result from the db for each query?